src/test/integration/IntegrationBase.t.sol

```diff
@@ -640,8 +659,17 @@ abstract contract IntegrationBase is IntegrationDeployer {
         for (uint i = 0; i < tokens.length; i++) {
             uint prevBalance = prevTokenBalances[i];
             uint curBalance = curTokenBalances[i];
+            uint expected = prevBalance + addedTokens[i];
+            bool isError = expected != curBalance;
+
+            if (isError) {
+                console.log("\nCurrent `%s` balance != previous balance + added tokens".red(), _getTokenName(tokens[i]));
+                console.log("   Previous Balance:", _toStringWad(prevBalance));
```

Comment:
beacon chain shares are in terms of gwei, so 1e9. Assume this value will appear smaller than it should since we're logging in terms of wei(wad)??